src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -497,15 +503,22 @@ function SearchAutocompleteList({
                 }));
             }
             case CONST.SEARCH.SYNTAX_FILTER_KEYS.HAS: {
-                const filteredHasValues = Object.values(CONST.SEARCH.HAS_VALUES).filter((hasValue) => {
-                    return hasValue.toLowerCase().includes(autocompleteValue.toLowerCase()) && !alreadyAutocompletedKeys.includes(hasValue.toLowerCase());
+                const filteredHasValues = hasAutocompleteList.filter((hasValue) => {
+                    return hasValue.value.toLowerCase().includes(autocompleteValue.toLowerCase()) && !alreadyAutocompletedKeys.includes(hasValue.value.toLowerCase());
                 });
 
                 return filteredHasValues.map((hasValue) => ({
                     filterKey: CONST.SEARCH.SEARCH_USER_FRIENDLY_KEYS.HAS,
-                    text: hasValue,
+                    text: hasValue.value,
                 }));
             }
+            case CONST.SEARCH.SYNTAX_FILTER_KEYS.IS: {
+                const filteredIsValues = isAutocompleteList.filter((isValue) => {
+                    return isValue.toLowerCase().includes(autocompleteValue.toLowerCase()) && !alreadyAutocompletedKeys.includes(isValue.toLowerCase());
+                });
+
+                return filteredIsValues.map((isValue) => ({filterKey: CONST.SEARCH.SYNTAX_FILTER_KEYS.IS, text: isValue}));
```

Comment:
Any reason we're not using `SEARCH_USER_FRIENDLY_KEYS` for `IS` like we do above for `HAS` ?

```suggestion
                return filteredIsValues.map((isValue) => ({filterKey: CONST.SEARCH.SEARCH_USER_FRIENDLY_KEYS.IS, text: isValue}));
```

Reply:
Updated